Remove bashisms from spectrum shell scripts

henrirosten opened the pull request:
This commit removes bashisms from spectrum shell scripts. This change is needed to be able to use the scripts from POSIX-compliant shells which are not bash compatible - such as dash.

Signed-off-by: Henri Rosten <henri.rosten@unikie.com>

vilvo commented:
Thanks Henri - this belongs to upstream improvements and review. We aim to minimize the delta to spectrum upstream for easier rebase. Please see https://spectrum-os.org/contributing.html

henrirosten committed:
Remove bashisms from spectrum shell scripts

This commit removes bashisms from spectrum shell scripts. This change is
needed to be able to use the scripts from POSIX-compliant shells which
are not bash compatible - such as dash.

Signed-off-by: Henri Rosten <henri.rosten@unikie.com>

henrirosten commented:
Merged upstream: https://spectrum-os.org/lists/hyperkitty/list/devel@spectrum-os.org/thread/6Z2DTZ2ZR7I3BHYNMOO3JGZ27OVWTXGA/

Therefore, closing this PR
